.changes/fix-ipc-payload.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri": 'patch:bug'
+---
+
+No longer unpacking and flattening the `payload` over the IPC so that commands with an argument called `options` don't break.
```

Comment:
It's actually also parameters named `error`, `cmd`, or 'callback`

Reply:
Updated